packages/specs/schema/src/components/default/objectMapper.ts

```
@@ -12,19 +13,23 @@ import {execMapper, registerJsonSchemaMapper} from "../../registries/JsonSchemaM
           ...options,
-          groups: input?.$forwardGroups || value?.$forwardGroups ? options.groups : undefined
+          parent: value,
+          groups: input?.get?.(VendorKeys.FORWARD_GROUPS) || value?.get?.(VendorKeys.FORWARD_GROUPS) ? options.groups : undefined
         };
 
-        // remove groups to avoid bad schema generation over children models
         obj[key] = execMapper("item", [value], opts);
         obj[key] = execMapper("nullable", [obj[key], value], opts);
+
+        if (value.isGeneric && obj[key]?.type) {
+          delete obj[key].type;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)